New package: compuphase.pawn version 4.1.7152

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #156108

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156140)

Dragon1573 committed:
New package: compuphase.pawn version 4.1.7152

Signed-off-by: 是蓝兔哟～ <49941141+Dragon1573@users.noreply.github.com>

stephengillie commented:
Automatic Validation ended with: 
>  The file was not found. Verify git path must match with the casing of the file in disk. Not found file on 'manifests\c\compuphase\pawn\4.1.7152\compuphase.pawn.installer.yaml'

stephengillie commented:
@wingetbot run

Trenly commented on manifests/c/compuphase/pawn/4.1.7152/compuphase.pawn.installer.yaml:
```diff
@@ -0,0 +1,18 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: compuphase.pawn
```
```suggestion
PackageIdentifier: Compuphase.Pawn
```
The validation is failing because the publisher `Compuphase` with a capital C already exists. This manifest will need to be put under the existing folder

Trenly commented on manifests/c/compuphase/pawn/4.1.7152/compuphase.pawn.locale.en-US.yaml:
```diff
@@ -0,0 +1,20 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: compuphase.pawn
```
```suggestion
PackageIdentifier: Compuphase.Pawn
```
The validation is failing because the publisher `Compuphase` with a capital C already exists. This manifest will need to be put under the existing folder

Trenly commented on manifests/c/compuphase/pawn/4.1.7152/compuphase.pawn.yaml:
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: compuphase.pawn
```
```suggestion
PackageIdentifier: Compuphase.Pawn
```
The validation is failing because the publisher `Compuphase` with a capital C already exists. This manifest will need to be put under the existing folder

Trenly commented on manifests/c/compuphase/pawn/4.1.7152/compuphase.pawn.locale.en-US.yaml:
```diff
@@ -0,0 +1,20 @@
+PackageLocale: en-US
+Publisher: Thiadmer Riemersma
+PublisherUrl: https://github.com/compuphase
+PackageName: Pawn
```
```suggestion
PackageName: Pawn embedded scripting language
```

Trenly commented on manifests/c/compuphase/pawn/4.1.7152/compuphase.pawn.locale.en-US.yaml:
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: compuphase.pawn
+PackageVersion: 4.1.7152
+PackageLocale: en-US
+Publisher: Thiadmer Riemersma
```
```suggestion
Publisher: CompuPhase
```
This should match control panel

Dragon1573 committed:
Apply suggestion from code review

- Change package identifier
- Change package name in order to match ARP Entry
- Change publisher info in order to match ARP Entry

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

stephengillie commented:
Verify:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/4733b788-a718-4112-bb37-44f38c960be2)
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/d9c94947-20e5-40c4-8b66-70f1a9a66644)
